CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -297,6 +297,9 @@ CF_EXPORT CFStringRef CFCopyFullUserName(void) {
     uid_t euid;
     __CFGetUGIDs(&euid, NULL);
     struct passwd *upwd = getpwuid(euid ? euid : getuid());
+#ifdef __ANDROID__
```

Comment:
I would like to use a `TARGET_OS_ANDROID` value instead, if possible, to match the rest of our macros.

Reply:
Sure.